lib/Parser/Parse.h

```diff
@@ -195,6 +195,7 @@ class Parser
     bool                m_hasParallelJob;
     bool                m_doingFastScan;
     int                 m_nextBlockId;
+    charcount_t         m_lastRParen;
```

Comment:
Needs to flow out of ParseExpr but can be passed as optional to both. I didn't do this initially because I thought it might be hard to pass the optional param down all paths but actually seems to be pretty simple.